config.yaml

```
@@ -123,7 +125,20 @@ groot:
     maxlength: 125           # Maxlength for groot index
     covcutoff: 0.97          # Coverage cutoff for groot report
     lowcov: False            # Report ARGs with no 5' or 3' coverage. Overrides covcutoff.
-
+amrplusplus:
+    databases:
+        db: "scripts/amrplusplus/db/megares_modified_database_v2.00.fasta"
```

Perhaps rename this to `megares` instead of `db`, because the key is now `config["amrplusplus"]["databases"]["db"]` which is not very informative.
```suggestion
        megares_fasta: "scripts/amrplusplus/db/megares_modified_database_v2.00.fasta"
```

---

In addition, we should discuss whether we want to store database-type data in the `scripts` folder. I would argue we should put this in the `dbdir` folder. However, this introduces a fun challenge: referencing a variable defined in the same YAML-file :D. Another alternative is that we just hardcode the `dbdir` as `databases`, but this requires careful consideration so we don't break anything (`dbdir` is an old legacy)

---

I havent yet seen `dbdir` in action but it is possible to during a run, create a folder called `databases` and just download it there. Im not myself a fan of keeping the databases in the scripts directory anyway.

---

I think `dbdir` is no longer in use by any part of the codebase. We could hardcode all databases to download into `databases/{dbname}`. Try it and let's see how we can do this. When we reach a final decision we have to remember to update the docs.